runbenchmark.py

```diff
@@ -8,13 +8,15 @@
 # prevent asap other modules from defining the root logger using basicConfig
 import amlb.logger
 
+from openml.config import get_cache_directory
+
 import amlb
 from amlb.utils import Namespace as ns, config_load, datetime_iso, str2bool, str_sanitize, zip_path
 from amlb import log, AutoMLError
 
 
 default_dirs = ns(
-    input_dir="~/.openml/cache",
+    input_dir=get_cache_directory(),
```

Comment:
should be good enough to fix `https://github.com/openml/automlbenchmark/issues/321` and I don't see any negative impact as in AWS mode, the input_dir is always overriden.

Reply:
I am a bit confused, that seems to be what I proposed in the original post of the issue. Why are you in favor now?

Reply:
was it? I was confused myself when making this change, so I checked the ticket to recall my arguments then, and it looks like I was mainly against having 2 folders: one for input and one for the openml cache…
Maybe there was a mistunderstanding, idk, I've been searching what could go wrong with this fix, and now I don't see anything.

Reply:
I think the issue got derailed. The original proposal was exactly this. However I then asked why it wasn't this already, and proposed that perhaps it is due to our own split files, in which case I said maybe a second folder could be an option if that's why it was separated. Then the discussion continued about the second directory as opposed to changing the default directory 😅 

Still, I do think there's something to be said against creating files in the openml cache directory itself, but for me that wouldn't be a strong enough incentive not to share the cache directory.

Reply:
Ok, let's do this for now: it should make the majority happy.